staging/src/k8s.io/kubectl/pkg/cmd/config/view.go

```diff
@@ -96,6 +95,7 @@ func NewCmdConfigView(streams genericclioptions.IOStreams, ConfigAccess clientcm
 	cmd.Flags().BoolVar(&o.RawByteData, "raw", o.RawByteData, "Display raw byte data")
 	cmd.Flags().BoolVar(&o.Flatten, "flatten", o.Flatten, "Flatten the resulting kubeconfig file into self-contained output (useful for creating portable kubeconfig files)")
 	cmd.Flags().BoolVar(&o.Minify, "minify", o.Minify, "Remove all information not used by current-context from the output")
+	cmd.Flags().BoolVar(&o.ShowSecrets, "show-secrets", o.ShowSecrets, "Display sensitive data")
```

Comment:
Why we are introducing new flag?. As far as I understand from the issue, `--raw` will show sensitives and others will not? or none will not show any sensitive?

Reply:
This flag was added because, due to the verbiage of the flag help text for `--raw`, it is not immediately apparent to the user that `--raw` will be displaying secrets. It seemed to me like the best option was to add a new flag to explicitly display secrets, because there may be situations where users will want to display the raw byte data of their client certificate data, but not their client key data as that is a secret. Admittedly this is a bit of a breaking change in terms of user expectation.

Reply:
Do we think there will be a path forward to move to using both of these flags? I think it is important to give users the option to view their raw bytes with out forcing them to also expose their secrets. I know it's a relatively small threat vector but it feels like an easy enough gap to close as long as we do it right without breaking things for people unexpectedly.

Reply:
Thinking about this more, wonder if we can create two new flags, `--show-bytes` and `--show-secrets`, note that `--raw` will perform the function of both in the help text as that matches the current functionality of the flag, and either plan a deprecation for `--raw` or just leave `--raw` as a way for users to be able to functionally pass both flags at once.

Reply:
Let's keep it simple, I bet you can easily figure out additional 2 or 3 flags for that command, but unless we'll have a direct feedback from users I'd prefer to keep it the way it is :wink:

Reply:
Alright removed new flags.